src/main/resources/db/migration/claimstore/V2019_02_15_1328__Add_claim_document_store_column.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE claim
+  ADD COLUMN claim_document_store jsonb NULL;
```

Comment:
Prefix `claim` looks unnecessary here. How about renaming it `documents`?

Reply:
because these documents belong to the claim I have named the column as such.